internal/services/network/subnet_resource.go

```diff
@@ -91,9 +92,9 @@ var subnetDelegationServiceNames = []string{
 	"Microsoft.Web/hostingEnvironments",
 	"Microsoft.Web/serverFarms",
 	"NGINX.NGINXPLUS/nginxDeployments",
+	"Oracle.Database/networkAttachments",
```

Comment:
Thank you so much for fixing the ordering here ❤️

Reply:
I cannot believe that you guys actually merged my request for Informatica. Amazing. Thanks a lot!!!